.travis.yml

```diff
@@ -58,8 +59,9 @@ cache:
 
 before_install:
   - ./tools/travis/docker.sh
-  - pip install --upgrade pip setuptools six
-  - pip3 install --upgrade pip setuptools six
+  - sudo apt-get install -y python3-pip
```

Comment:
@neerajmangal Does the build still fails with `--upgrade` flag as current pip version seems to be old

```
You are using pip version 8.1.1, however version 19.2.3 is available.
```

Reply:
@dgrove-oss do these help on your pr?

Reply:
I didn't have time to finish yesterday; this one looks closer than mine.  I will close mine in favor of this one.

Reply:
> @neerajmangal Does the build still fails with `--upgrade` flag as current pip version seems to be old
> 
> ```
> You are using pip version 8.1.1, however version 19.2.3 is available.
> ```

It is fixed now using --upgrade.